WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -39,3 +47,98 @@ class Feature: NSObject {
         return feature.enabled
     }
 }
+
+
+extension FeatureFlag: CustomStringConvertible {
```

Comment:
Correct me if I'm wrong, but I understood that we need to conform to `CustomStringConvertible` because of `OverrideableFlag`, right?

In that case, wouldn't it make more sense to move the `description` to the `extension FeatureFlag: OverrideableFlag`?

Reply:
Sure, that makes sense 👍